extensions/gitpod-web/src/extension.ts

```diff
@@ -5,25 +5,20 @@
 /// <reference path='../../../src/vscode-dts/vscode.d.ts'/>
 /// <reference path='../../../src/vscode-dts/vscode.proposed.resolvers.d.ts'/>
 
-import * as workspaceInstance from '@gitpod/gitpod-protocol/lib/workspace-instance';
-import * as grpc from '@grpc/grpc-js';
 import * as fs from 'fs';
 import * as os from 'os';
 import * as uuid from 'uuid';
 import { GitpodPluginModel, GitpodExtensionContext, setupGitpodContext, registerTasks, registerIpcHookCli } from 'gitpod-shared';
 import { GetTokenRequest } from '@gitpod/supervisor-api-grpc/lib/token_pb';
-import { PortsStatus, ExposedPortInfo, PortsStatusRequest, PortsStatusResponse, PortAutoExposure, PortVisibility, OnPortExposedAction } from '@gitpod/supervisor-api-grpc/lib/status_pb';
```

Comment:
let's revert changes to this file

good PR should make as less a possible to this file to avoid breaking existing logic